dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/converter/EventTrackerConverterService.java

```diff
@@ -218,6 +218,7 @@ private ProgramStageInstance from( TrackerPreheat preheat, Event event, ProgramS
             programStageInstance.setStoredBy( event.getStoredBy() );
         }
         programStageInstance.setLastUpdated( now );
+        programStageInstance.setDeleted( false );
```

Comment:
In the import logic we cannot update a tracker object that is soft deleted and we cannot update deleted to false because this can be done only in the delete endpoint.
Make it explicit should make it clear that this is a read-only value and if there is a value coming from the payload is just ignored.